### PR TITLE
Created Load balancer

### PR DESCRIPTION
**Pull request by agoel2411**

### CODE REVIEW

File: level2/load_balancer.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+resource "aws_lb_listener" "main" {
+    load_balancer_arn = aws_lb.main.arn
+    port              = "80"
+    protocol          = "HTTP"
+
+    default_action {
+        type = "forward"
+        target_group_arn = aws_lb_target_group.main.arn
+    }
+}
```
Comment (geoffrey1330) [inline level2/load_balancer.tf:71]: could you please, add a newline at the end of file.

File: level2/instance.tf [hunk 2]
```diff
@@ -18,11 +18,11 @@ resource "aws_instance" "public" {
   ami                         = data.aws_ami.amazonlinux.id
   associate_public_ip_address = true
   instance_type               = "t3.micro"
-  key_name                    = "main"
+  key_name                    = "poc"
   vpc_security_group_ids      = [aws_security_group.public.id]
   subnet_id                   = data.terraform_remote_state.level1.outputs.public_subnet_id[1]
 
-  user_data = file("user-data.sh")
+
```
Comment (geoffrey1330) [inline level2/instance.tf:25]: too much line space. could you please reduce it to one

**Review by geoffrey1330 — CHANGES_REQUESTED** [review]

File: level2/load_balancer.tf (hunk 1 above)
Comment (agoel2411) [reply level2/load_balancer.tf]: done

File: level2/instance.tf (hunk 2 above)
Comment (agoel2411) [reply level2/instance.tf]: done

**Review by geoffrey1330 — APPROVED** [review]

**Pull request merged**